wordpress-mobile/WordPress-iOS
# [Dark Mode] General Improvements

Refs. #12320 

This PR fixes some color for:
_• Edit comment is white text on white background.
• Pending comments have a white background. / Comment moderation has incorrect background on the comment cell, screenshot added to the thread.
• Gravatar cropping screen has a white background_
![dark-mode-comments-gravatar-dark](https://user-images.githubusercontent.com/912252/65692032-c999d280-e069-11e9-9f9c-ae651a696663.jpg)
![dark-mode-comments-gravatar-light](https://user-images.githubusercontent.com/912252/65692034-c999d280-e069-11e9-86cb-10733428f2de.jpg)

_• Map in Stats DWMY: I changed the map fill color using `.systemGray4` and fixed the stroke color_
![dark-mode-stats-map](https://user-images.githubusercontent.com/912252/65692159-01a11580-e06a-11e9-9ddc-fe07eb877f27.jpg)

## To test:
• Run this branch with an iOS 13 device
• Open a comment in pending to check the colors and switch between light and dark mode
• Edit a comment to check the colors and switch between light and dark mode
• Open Stats->DWMY and scroll till the map to check the colors and switch between light and dark mode
• Tab Me, open on the user picture and its cropping screen to check the colors and switch between light and dark mode
• Test everything works with iOS 12

Update release notes:
